marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/models/createModel.sjs

```diff
@@ -21,11 +21,14 @@ xdmp.securityAssert("http://marklogic.com/data-hub/privileges/write-entity-model
 
 const httpUtils = require("/data-hub/5/impl/http-utils.sjs");
 const entityLib = require("/data-hub/5/impl/entity-lib.sjs");
+const hubUtils = require("/data-hub/5/impl/hub-utils.sjs");
 
 var input = fn.head(xdmp.fromJSON(input));
 
 const name = input.name;
 const description = input.description;
+const namespace = input.namespace ? input.namespace : null;
```

Comment:
If we do end up adding these 2 inputs, I think we should turn this into a JsonNode that gets passed to the DS endpoint.